### ISSUE TITLE
Missing aggregation options for new bar chart panels

### ISSUE THREAD

**Issue by nathanpainchaud** [comet-ml/issue-tracking#496]
## Describe the Bug
The bar chart panel is missing aggregation options (e.g. min, max, mean, etc.) that were available until recently (i.e. a few weeks ago at most). Depending on why these aggregation options were removed in the first place, this might be a feature request rather than a bug report. But given that the option was there recently, I consider a feature regression.

## Expected behavior
It should be possible to aggregate metrics in bar chart panels using more than _first_, _last_ and _count_ methods. As an example, the scatter chart panel offers a lot more aggregation options: _count_, _sum_, _avg_, _median_, etc.

## Where is the issue?
- [ ] Comet Python SDK
- [x] Comet UI
- [ ] Third Party Integrations (Huggingface, TensorboardX, Pytorch Lighting etc)

## To Reproduce
1. Go to a project and try to a add a new bar chart panel
2. Click on the _Aggregation_ drop-down list to see the (limited) aggregation options

## Screenshots or GIFs
List of aggregations available in bar charts:
![bar_chart_agg](https://user-images.githubusercontent.com/23144457/214571735-4808ada6-70a6-4693-9b7c-f1aea822a14c.png)

For comparison, (partial) list of aggregations available in scatter charts:
![scatter_chart_agg](https://user-images.githubusercontent.com/23144457/214572645-b77e5a28-faeb-4a25-92cc-b8c6deeeeea5.png)

**Comment by ferc** [comment]
Hello @nathanpainchaud , once you select a metric, if it's a numeric metric it'll display the rest of the aggregations (since the bar chart now could accept not-numeric metrics/parameters, this behavior in the select changed, but if you find it confusing we can work on improving the UX here)
<img width="506" alt="Screenshot 2023-01-25 at 2 18 51 PM" src="https://user-images.githubusercontent.com/902031/214573554-9512d8ac-9c5f-41b4-a3c1-71e594250076.png">

Let me know if this helps you, have a nice day!

**Comment by ferc** [comment]
@nathanpainchaud I have 2 theories that could bring to detect a metric as a `string` when it's not: 
1. One of your experiments logged this metric as a string
2. One of your experiments logged an invalid number (`NaN`), therefore it could be mislead and interpreted as a string

Hope it helps

**Comment by nathanpainchaud** [comment]
@ferc Thanks for the suggestion. However, I really don't they're likely/possible in my case. The metric I log is the loss used to train my models, so it being a string or a `NaN` would not happen silently without visibly affecting the training.

Do you have any more possible theories as to why a metric would not be interpreted as numeric for the aggregation? Especially given that the preview on the right seems to show that the UI of the bar graph itself interprets the metric perfectly well as numeric.
